readme.md

```diff
@@ -3,90 +3,199 @@ Penn Labs Constitution
+
+#### Responsibilities
+
+Penn Labs is responsible for maintaining mission critical applications for the student
+body. Being a part of the Penn Labs community is a large responsibility as well as a
+privelege with benefits, and general members are responsible for the following: 
+
+- Attend GBMs as well as any team-specific meetings, notifying their Team Lead of any expected 
+absences.
+- Contribute to the success of the product team to which they are a part.
+
+
+## Compensation
+
+Penn Labs is committed to fostering a community among its members and making sure they know that
+their work is valued in the Penn community. In order to do this while maintaining the flexibility
+to grow the organization, Penn Labs commits to spending the majority of its budget on items and
+activities that directly engage and benefit its membership, including but not limited to:
+
+- Educational resources (e.g. technical manuals, online courses)
+- Team building retreat
+- Meals at GBMs
+- Subsidized Apparel
+
+
+## Stakeholders
+
+### Penn Labs Board
+
+The Penn Labs Board is intended to oversee the organization.  Comprised of key stakeholders and
+funding partners, it will meet at least once every semester to receive a progress report and
+approve ongoing and future Penn Labs activities. Though the board will be compelled to meet as
+described above, voting can be done online over e-mail.
+
+#### Penn Labs Board Membership
+- One Undergraduate Assembly representative, which will be designated as the UA President by default
+  with the option to delegate their voting rights to any Executive Board member of the UA.
+- One representative from the Provost’s Office.
+- One representative from OSA.
+- One representative from ISC.
+- One representative from the Office of the University Registrar.
+- Penn Labs Directors will share a vote.
+- Non-voting members:
+  - The Undergraduate Assembly Secretary or an Online Services Coordinator from the Undergraduate
+    Assembly if found necessary, charged with helping to set agendas, liaise with administrators, be
+    invested in Penn Labs’ success, etc.
 
 
 ## Funding Sources
 
 Penn Labs receives a budget from the Undergraduate Assembly to be matched by the Provost’s
-office for the school year. Penn Labs conducts payroll through the Provost’s office.
-
-Though Penn Labs will partner with a variety of student groups on campus, funding for Penn
-Labs will remain independent of projects, leaving the group to work on the highest-impact products
-for student life, without regard to a particular student group’s ability to raise funding.
-Penn Labs cannot maintain a client-provider relationship with any student group. Proposals like
-payment per-project or conditional payment based on project completion are against the non-profit
-nature of the work and not considered potential funding sources.
+office for the school year.
 
 Penn Labs is open to receive donations beyond its base funding on an ad-hoc basis without
```

Comment:
We should probably clarify that this doesn't prevent Labs from taking on contracted work.